### PR TITLE
add beelzebub.yml

### PR DESCRIPTION
**Pull request by mariocandela**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

**Review by Ki-er — CHANGES_REQUESTED** [review]
Tags act as categories rather than traditional tags.

**Comment by mariocandela** [comment]
> Tags act as categories rather than traditional tags.

Thanks for the review, issue resolved.

**Review by Ki-er — APPROVED** [review]
LGTM!

File: software/beelzebub.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: "beelzebub"
+website_url: "https://beelzebub-honeypot.com/"
+source_code_url: "https://github.com/mariocandela/beelzebub"
+description: "A secure low code honeypot framework, leveraging AI for System Virtualization."
+licenses:
+  - MIT License
```
Comment (nodiscc) [inline software/beelzebub.yml:6]:
```suggestion
  - MIT
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: software/beelzebub.yml (hunk 1 above)
Comment (mariocandela) [reply software/beelzebub.yml]: Thanks for the review, issue resolved.

**Comment by mariocandela** [comment]
@nodiscc @Ki-er If it's okay now, shall we bring it? Thank you

File: software/beelzebub.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: "beelzebub"
+website_url: "https://beelzebub-honeypot.com/"
+source_code_url: "https://github.com/mariocandela/beelzebub"
+description: "A secure low code honeypot framework, leveraging AI for System Virtualization."
```
Comment (nodiscc) [inline software/beelzebub.yml:4]:
```suggestion
description: "Secure low code honeypot framework, leveraging AI for System Virtualization."
```

Minor syntax standardization

File: software/beelzebub.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+licenses:
+  - MIT
+platforms:
+  - Docker
+  - K8S
+  - Go
+tags:
+  - Miscellaneous 
+depends_3rdparty: false
```
Comment (nodiscc) [inline software/beelzebub.yml:13]:
```suggestion
depends_3rdparty: false
```

> utilizes virtualization techniques powered by OpenAI Generative Pre-trained Transformer.

Does it depend on OpenAI services? If so, `depends_3rdparty: yes` must be set

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: software/beelzebub.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+name: "beelzebub"
+website_url: "https://beelzebub-honeypot.com/"
+source_code_url: "https://github.com/mariocandela/beelzebub"
+description: "Secure low code honeypot framework, leveraging AI for System Virtualization."
```
Comment (nodiscc) [inline software/beelzebub.yml:4]:
```suggestion
description: "Honeypot framework designed to provide a highly secure environment for detecting and analyzing cyber attacks.
```

The description should explain what the software does.

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: software/beelzebub.yml (hunk 4 above)
Comment (mariocandela) [reply software/beelzebub.yml]: Thanks @nodiscc, resolved

**Review by nodiscc — APPROVED** [review]

**Comment by mariocandela** [comment]
Hi @nodiscc, 

What do you think about merging it? 

Let me know if you have anything else to add.

Thank you

**Pull request merged**